dftimewolf/lib/state.py

```diff
@@ -52,27 +52,28 @@ def load_recipe(self, recipe):
       module = self.config.get_module(module_name)(self)
       self._module_pool[module_name] = module
 
-  def store_data(self, key, data):
+  def store_data(self, data):
     """Thread-safe method to store data in the state's store.
 
     Args:
-      key: Key in which to store the data.
-      data: The data to store.
+      data (containers.interface.AttributeContainer): The data to store.
     """
     with self._store_lock:
-      self.store[key] = data
+      self.store.append(data)
 
-  def get_data(self, key):
+  def get_data(self, container):
```

Comment:
Rename to container_class, if this indeed meant to be a class, and not an instance.